docs/wallets/ledger_desktop_wallet.md

```diff
@@ -16,23 +16,25 @@ In case you have any enquiry about using Ledger device with desktop wallet, kind
 
 
 ## Install the Crypto.org Chain desktop wallet app for Ledger and create the wallet  
-We will use Ledger Nano S for this illustration:
+
 ### Step 1 - Install the Crypto.org Chain app on your Ledger device
 
 - **Step 1.1** - Connect and unlock your Ledger device
 
 - **Step 1.2** - On your Ledger device, go to "*Install app*", It should show "*Open Ledger live to install apps*"
 
-- **Step 1.3** - On the Ledger Live, find and install Crypto.org Chain in the app catalog under "*Manger*" as below:
-        <img src="./assets/ledger_desktop_wallet/s1-4.png" />
+- **Step 1.3** - On the Ledger Live, find Crypto.org Chain & Ethereum to install in the app catalog under "*Manager*" as below:
+        <img src="./assets/ledger_desktop_wallet/s1-4-1.png" />
```

Comment:
Need to adjust/delete the imgs with mismatched names with those at https://github.com/crypto-org-chain/chain-docs/tree/master/docs/wallets/assets/ledger_desktop_wallet